apps/frontend/package.json

```diff
@@ -39,7 +39,6 @@
     "zod": "^3.23.8"
   },
   "devDependencies": {
-    "@types/node": "^20",
```

Comment:
I may add this back, I was trying to see if the node import was causing the conflict with localStorage (there's a ts localStorage and a node localStorage)